fuzz/Cargo.toml

```diff
@@ -102,3 +119,17 @@ name = "instantiate-many"
 path = "fuzz_targets/instantiate-many.rs"
 test = false
 doc = false
+
+[[bin]]
+name = "dynamic_component_api"
+path = "fuzz_targets/dynamic_component_api.rs"
+test = false
+doc = false
+required-features = ["component-model"]
+
+[[bin]]
+name = "static_component_api"
+path = "fuzz_targets/static_component_api.rs"
+test = false
+doc = false
+required-features = ["component-model"]
```

Comment:
Currently the way time allotment on oss-fuzz works is that we're given a fixed time slice to balance between all of our fuzzing executables. In that sense the more fuzzers we have the more they take time away from other fuzzers. Since these are fuzzing the same thing in spirit could these two fuzzers be folded into one? The fuzz input could be used to determine whether the dynamic or static API could be fuzzed then I think.